guardian/scribe-plugin-noting
# Fixing issues created by #40

This PR should fix the following issues:

https://github.com/guardian/scribe-plugin-noting/issues/41
https://github.com/guardian/scribe-plugin-noting/issues/42
https://github.com/guardian/scribe-plugin-noting/issues/44
https://github.com/guardian/scribe-plugin-noting/issues/45

### QA
Try splitting a note into two and then un-note the first note. This should un-note ONLY the first note and leave the second untouched.

Also split one note into to and check they both have distinct data-note-id's
